aucoalesce/testdata/ubuntu-16.10-linux-4.8.0.json.golden

```diff
@@ -481,7 +526,16 @@
           "type": [
             "start"
           ]
-        }
+        },
+        "user": {
+          "id": "unset",
+          "effective": {
+            "name": "(invalid user)"
```

Comment:
does it make sense to actually enrich these if they're invalid/unset?

Reply:
This is a failed ssh login attempt using an invalid user, I think it's worth to report that back to the user instead of an empty user.